### PR TITLE
Task/csi 5092 creation of second volume replication fails

### PR DESCRIPTION
**Pull request by ELENAGER**

### CODE REVIEW

File: controllers/tests/controller_server/addons_server_test.py [hunk 1]
```diff
@@ -130,9 +130,15 @@ def test_enable_replication_no_volume_fails(self):
...
-                                            grpc_status=grpc.StatusCode.FAILED_PRECONDITION)
+        replication_request = self._prepare_request_params(replication_type=REPLICATION_TYPE_EAR,
+                                                           replication_id="")
+        self._prepare_replication_mocks()
+
+        self.servicer.EnableVolumeReplication(self.request, self.context)
+
+        self.assertEqual(grpc.StatusCode.FAILED_PRECONDITION, self.context.code)
+        self.mediator.get_replication.assert_called_once_with(replication_request)
+        self.mediator.create_replication.assert_not_called()
```
Comment (ArbelNathan) [inline controllers/tests/controller_server/addons_server_test.py:141]: can you add idempotency test?

**Review by matancarmeli7 — DISMISSED** [review]
LGTM

File: controllers/tests/controller_server/addons_server_test.py (hunk 1 above)
Comment (ELENAGER) [reply controllers/tests/controller_server/addons_server_test.py:148]: I added test for the EAR + idempotency (see test_enable_ear_replication_idempotency_succeeds)

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**